src/config/util.swift

```diff
@@ -175,14 +175,23 @@ func readJSON(_ file: URL) -> JSON? {
 
 func openInEditor(_ path: String) {
   let apps = ["VSCodium", "Visual Studio Code"]
+  let fileURL = URL(fileURLWithPath: path)
```

Comment:
Given it's actually the URL that is needed, let's change the function's signature to accept fileURL, and modify its all call sites?